lib/cosmos/io/json_drb_object.rb

```diff
@@ -36,7 +38,7 @@ class JsonDRbObject
     def initialize(hostname, port, connect_timeout = 1.0)
       hostname = '127.0.0.1' if (hostname.to_s.upcase == 'LOCALHOST')
       begin
-        @addr = Socket.pack_sockaddr_in(port, hostname)
+        addr = Socket.pack_sockaddr_in(port, hostname)
```

Comment:
I don't see addr being used anywhere. What is the point of this line?

Reply:
Yeah, addr isn't used, this line is purely there because pack_sockaddr_in will raise an exception and allow me to catch an invalid hostname at initialize time.  If I don't do this, I don't catch the invalid host until I try to post something.  That'd drive changes to things that use JsonDRbObject.